package.json

```diff
@@ -90,7 +90,7 @@
     "babel-eslint": "^9.0.0",
     "babel-polyfill": "^6.26.0",
     "chai": "^4.2.0",
-    "eslint": "^5.5.0",
+    "eslint": "^6.2.1",
```

Comment:
Please upgrade the version of `@folio/eslint-config-stripes` since you here to be `"@folio/eslint-config-stripes": "^5.0.0". 

It causes not a lot of changes and would improve stuff like tests by removing [describe.only](https://github.com/folio-org/ui-search/blob/master/test/bigtest/tests/search-results-test.js#L16).

```
/ui-search/src/Filters/Filters.js
  125:7  error  Prefer fragment shorthand over React.Fragment  react/jsx-fragments

/ui-search/src/Search/Search.js
   94:16  warning  'filterName' is defined but never used                            no-unused-vars
  111:16  warning  'index' is defined but never used                                 no-unused-vars
  122:16  warning  'index' is defined but never used                                 no-unused-vars
  174:61  error    Use an object spread instead of `Object.assign` eg: `{ ...foo }`  prefer-object-spread
  179:18  warning  'index' is defined but never used                                 no-unused-vars

/ui-search/src/Search/model/redirectParams.js
  2:1  error  Too many blank lines at the beginning of file. Max of 1 allowed  no-multiple-empty-lines

/ui-search/src/ViewRecord.js
  26:5  error  Prefer fragment shorthand over React.Fragment  react/jsx-fragments

/ui-search/test/bigtest/interactors/search.js
  1:1  error  File has too many classes (3). Maximum allowed is 1  max-classes-per-file

/ui-search/test/bigtest/tests/search-results-test.js
  16:10  error  describe.only not permitted  no-only-tests/no-only-tests

✖ 10 problems (6 errors, 4 warnings)
  4 errors and 0 warnings potentially fixable with the `--fix` option.
```